imap_processing/cdf/utils.py

```diff
@@ -56,74 +50,31 @@ def write_cdf(
     ----------
         data : xarray.Dataset
             The dataset object to convert to a CDF
+        filepath: Path
+            The output path, including filename, to write the CDF to.
         descriptor : str
             The descriptor to insert into the file name after the
             orbit, before the SPICE field.  No underscores allowed.
```

Comment:
I think this could also be removed since it is not used?

Reply:
Merge fail 😆